howdy/debian/postinst

```diff
@@ -106,83 +106,6 @@ if "upgrade" in sys.argv:
 
 	sys.exit(0)
 
-log("Downloading dlib")
-
-dlib_archive = "/tmp/v19.16.tar.gz"
-loader = which("wget")
-LOADER_CMD = None
-
-# If wget is installed, use that as the downloader
-if loader:
-	LOADER_CMD = [loader, "--tries", "5", "--output-document"]
-# Otherwise, fall back on curl
-else:
-	loader = which("curl")
-	LOADER_CMD = [loader, "--retry", "5", "--location", "--output"]
-
-# Assemble and execute the download command
-cmd = LOADER_CMD + [dlib_archive, "https://github.com/davisking/dlib/archive/v19.16.tar.gz"]
-handleStatus(sc(cmd))
-
-# The folder containing the dlib source
-DLIB_DIR = None
-# A regex of all files to ignore while unpacking the archive
-excludes = re.compile(
-	"davisking-dlib-\w+/(dlib/(http_client|java|matlab|test/)|"
-	"(docs|examples|python_examples)|"
-	"tools/(archive|convert_dlib_nets_to_caffe|htmlify|imglab|python/test|visual_studio_natvis))"
-)
-
-# Open the archive
-with tarfile.open(dlib_archive) as tf:
-	for item in tf:
-		# Set the destenation dir if unset
-		if not DLIB_DIR:
-			DLIB_DIR = "/tmp/" + item.name
-
-		# extract only files sufficient for building
-		if not excludes.match(item.name):
-			tf.extract(item, "/tmp")
-
-# Delete the downloaded archive
-os.unlink(dlib_archive)
-
-log("Building dlib")
-
-cmd = ["sudo", "python3", "setup.py", "install"]
-cuda_used = False
-
-# Compile and link dlib
-try:
-	sp = subprocess.Popen(cmd, cwd=DLIB_DIR, stdout=subprocess.PIPE)
-except subprocess.CalledProcessError:
-	print("Error while building dlib")
-	raise
-
-# Go through each line from stdout
-while sp.poll() is None:
-	line = sp.stdout.readline().decode("utf-8")
-
-	if "DLIB WILL USE CUDA" in line:
-		cuda_used = True
-
-	print(line, end="")
-
-log("Cleaning up dlib")
-
-# Remove the no longer needed git clone
-del sp
-rmtree(DLIB_DIR)
-
-print("Temporary dlib files removed")
-
-log("Configuring howdy")
-
-# Manually change the camera id to the one picked
-for line in fileinput.input(["/etc/howdy/config.ini"], inplace=1):
-	line = line.replace("use_cnn = false", "use_cnn = " + str(cuda_used).lower())
-	print(line, end="")
-
```

Comment:
Why did you remove these lines?

Reply:
CNN depends on CUDA, and `nvidia-cuda-dev` is only a suggested package i guess. Could negatively impact performance by not enabling it automatically like this